### PR TITLE
Defect - Collapsible Regions 'Open By Default' Not Working Properly ++

### PR DESCRIPTION
**Pull request by nicholasnadel**
https://trello.com/c/vPoGxydD
https://dev-www.chapman.edu/test-section/nick-test/collapsible-improvements.aspx
https://dev-www.chapman.edu/test-section/nick-test/dmac-collapsible-fix.aspx

### CODE REVIEW

**Comment by mandythomas** [comment]
@nicholasnadel Why is there omniNav files mixed in this pull request?

**Comment by nicholasnadel** [comment]
> @nicholasnadel Why is there omniNav files mixed in this pull request?

The Two Column Template in the Modular Controller was still using an older Omninav design from a  previous dev team. I had to add the current production v2 design as a partial in order to test offsetting its height when toggling the accordions and scrolling to the respective accordion section

File: app/controllers/content_types/modular_controller.rb [hunk 1]
```diff
@@ -247,7 +247,7 @@ def two_column
         "SOCIAL ACCOUNTS" => "TODO: _cascade/formats/level/social_accounts",
 
         # TODO: convert these to cascade_format action.
```
Comment (nicholasnadel) [inline app/controllers/content_types/modular_controller.rb]: The Two Column Template in the Modular Controller was still using an older Omninav design from a previous dev team. I had to add the current production v2 design as a partial in order to test offsetting its height when toggling the accordions and scrolling to the respective accordion section

Comment (mandythomas) [reply app/controllers/content_types/modular_controller.rb]: ok makes sense

**Pull request merged**